/**:
  ros__parameters:
    grasp_height: 0.025